bin/madoc

```diff
@@ -141,6 +141,11 @@ case "${commandToRun}" in
         cp ./config/composer.json ./composer.json
         php ./config/patch-composer.php
         composer install --no-dev --no-scripts --optimize-autoloader
+
+        ## Sorting room
+        cd services/sorting-room;
```

Comment:
This would be better as: `{cd services/sorting-room; ./build.sh}` since it runs in a subshell, and you don't have to `cd` back out when the subshell exits.